driver/driver.go

```diff
@@ -63,39 +63,52 @@ var driverName = "sqlite3"
 
 func init() {
 	if driverName != "" {
-		sql.Register(driverName, sqlite{})
+		sql.Register(driverName, &SQLiteDriver)
 	}
 }
 
+// SQLiteDriver is a global Driver{} instance
+// registered under [database/sql] as "sqlite3".
+var SQLiteDriver = Driver{}
+
 // Open opens the SQLite database specified by dataSourceName as a [database/sql.DB].
 //
 // The init function is called by the driver on new connections.
 // The conn can be used to execute queries, register functions, etc.
 // Any error return closes the conn and passes the error to [database/sql].
 func Open(dataSourceName string, init func(*sqlite3.Conn) error) (*sql.DB, error) {
-	c, err := newConnector(dataSourceName, init)
+	d := Driver{Init: init}
+	c, err := d.OpenConnector(dataSourceName)
 	if err != nil {
 		return nil, err
 	}
 	return sql.OpenDB(c), nil
 }
 
-type sqlite struct{}
+type Driver struct {
```

Comment:
`Driver` and `SQLiteDriver` are weird names… outside the package they'll be `driver.Driver` and `driver.SQLiteDriver`.

Reply:
Do you have any alternative suggestions? Perhaps just `driver.SQLite{}`? :thinking: